test/sdn_migration/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.21 AS builder
```

```suggestion
FROM registry.ci.openshift.org/openshift/release:golang-1.22 AS builder
```
go mod declares 1.22 now